src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css

```diff
@@ -118,7 +118,7 @@
 }
 
 .paragraph3Wrapper {
-  margin-top: 25px;
+  margin-top: 50px;
 }
 
 .ctaTitleBlock {
```

Comment:
On the line just below this, we're setting a max-width. Could we remove this altogether? I think the reason this was here before was that we were trying to micro-optimize the line breaking of the text, but in the new layout, this should be wider than it used to be. It looks like the natural line breaking looks fine for the current copy, so I think we can just remove the max-width completely.